src/pages/settings/Settings.tsx

```diff
@@ -78,14 +86,15 @@ const Settings: FC = () => {
       );
       const value = getValue(configField);
 
-      const group = configField.key.split(".")[0];
-      const isNewGroup = lastGroup !== group;
-      lastGroup = group;
+      const isNewCategory = lastCategory !== configField.category;
+      lastCategory = configField.category;
 
       return {
         columns: [
           {
-            content: isNewGroup && <h2 className="p-heading--5">{group}</h2>,
+            content: isNewCategory && (
+              <h2 className="p-heading--5">{configField.category}</h2>
```

Comment:
This has already been assigned to the `lastCategory` variable at this point, right? So we should be allowed to do:
```suggestion
              <h2 className="p-heading--5">{lastCategory}</h2>
```

Reply:
I am hesitant for this change, from reading I'd stumble -- why print the lastCategory here and not the current one. Probably better to keep that variable just to figure out if we reached a category boundry.